claat/cmd/export.go

```diff
@@ -45,7 +45,8 @@ type CmdExportOptions struct {
 }
 
 // CmdExport is the "claat export ..." subcommand.
-func CmdExport(opts CmdExportOptions) {
+// It returns true if the command succeeded, false otherwise.
+func CmdExport(opts CmdExportOptions) bool {
```

Comment:
It feels more "go-like" to return an error here instead. You could use something like [go-multierror](https://github.com/hashicorp/go-multierror):

```golang
var errs *multierror.Error

for range args {
    if res.err != nil {
        errs = multierror.Append(errs, res.err)
    }
}

return errs.ErrorOrNil()
```

Reply:
Ooh, neat module. I'll give it a try.